R-packages/evalcast/R/baseline.R

```diff
@@ -25,7 +25,8 @@ baseline_forecaster <- function(df,
                                 incidence_period,
                                 ahead,
                                 geo_type) {
-  stopifnot(incidence_period %in% c("epiweek", "day"))
+  assert_that(incidence_period %in% c("epiweek", "day"),
+              msg="incidence_period must be epiweek or day.")
```

Comment:
Here I'd actually have `incidence_period` have a default value of `c("epiweek", "day")`, then have the function open with

```r
incidence_period <- match.arg(incidence_period)
```

This is the standard way R functions handle an argument that can be one of several values; check out, say, `help(cor)` for an example (the `method` argument). And `match.arg` will raise an appropriate error if the argument is the wrong type.